extensions/functions_datetime.yaml

```
@@ -48,7 +53,10 @@ scalar_functions:
             value: timestamp_tz
           - name: y
             value: interval_year
-        return: timestamp
+          - name: timezone
```

As below, I'm inclined to introduce a new variant as opposed to change the existing.

---

Also, I think we should probably make these timezone arguments constant values.

---

+1 on marking them as constant. @rok, that's just adding `constant: true` at the same level as the return field.

---

Added. Are you sure that is the right level?

---

Uh, no. Not sure what I was thinking there, sorry... :sweat_smile: Like this:

```suggestion
          - name: timezone
            constant: true
```

instead of the one I told you to add before.

---

Fixed :).